GameEngine/src/main/java/dk/sdu/se/f23/InVasion/gamestates/PlayState.java

```diff
@@ -33,24 +28,24 @@ public void init() {
         stage = new Stage();
         map = new MapPlugin();
         map.onEnable(gsm.getGameData(), gsm.getWorld());
-        
-        textButtonStyle = new TextButton.TextButtonStyle();
+
+        TextButton.TextButtonStyle textButtonStyle = new TextButton.TextButtonStyle();
         textButtonStyle.font = new BitmapFont();
         textButtonStyle.fontColor = Color.WHITE;
-        backToShopButton = new TextButton("go back to shop button", textButtonStyle);
-        backToShopButton.setPosition(1750,800);
-        backToShopButton.addListener(new InputListener(){
+        TextButton backToShopButton = new TextButton("go back to shop button", textButtonStyle);
```

Comment:
Hvis den her knap skal være der, så skal den ikke hedde 'go back to shop _button_